config/default.yaml

```diff
@@ -1,4 +1,32 @@
----
+
+appName: 'frontend'
+project: 'pcq'
+version: 'x.x-laptop'
+deployment_env: 'local'
+showSitemap: false
+
+service:
+  name: 'pcq-frontend'
+
+e2e:
+  frontendUrl: 'https://pcq.aat.platform.hmcts.net'
+  proxyServer: ''
+  proxyBypassList: ''
+  outputDirectory: './functional-output'
+  defaultEnvironmentNodeEnv: 'development'
+  waitForTimeoutValue: 5000
+  waitForActionValue: 100
+  runBasicTests: false
+
+saucelabs:
+  browser: 'chrome'
+  username: 'username'
+  key: 'privatekey'
+  tunnelId: 'reformtunnel'
+  waitForTimeout: 5000
+  smartWait: 5000
+
+
```

Comment:
These changes should really go in the testing.yaml config file. You can get the test scripts to use this by changing the node environment, e.g. `"test:smoke": "NODE_ENV=testing mocha test/smoke/healthcheck.js --timeout 30000 --reporter spec",`